scripts/hudson/narayana.sh

```diff
@@ -985,10 +993,10 @@ export ANT_OPTS="$ANT_OPTS $IPV6_OPTS"
 
 # run the job
 
-[ $NARAYANA_BUILD = 1 ] && build_narayana "$@"
-[ $AS_CLONE = 1 ] && clone_as "$@"
+[ $NARAYANA_BUILD = 1 ] && build_narayana "$@" # NB we should do this after building the AS since it takes longer
```

Comment:
I don't understand the comment but I don't think there is a functional change here yet but what I would say is building Narayana after the AS would be wrong because then the AS would not be able to see (and include) the built Narayana